File: api/src/main/java/com/mengyunzhi/springBootStudy/entity/School.java
package com.mengyunzhi.springBootStudy.entity;

import javax.persistence.*;
import java.util.List;

@Entity
public class School {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;


    @OneToMany
    private List<Klass> klass;

    public School() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: web-app/src/app/norm/entity/School.ts
/**
 * 学校实体
 */
export class School {
  id: number;
  name: string;

  /**
   * 构造函数
   * @param id id
   * @param name 名称
   */
  constructor(id: number, name: string) {
    this.id = id;
    this.name = name;
  }
}
